### PR TITLE
Feature update animals table

### PR DESCRIPTION
**Pull request by Graciano1997**
In this milestone, I did the following activities:

I used database transactions.
Modified and delete data in SQL.
Prepared complex queries that answered analytical questions.
update readme
Screenshot (183)
Screenshot (184)
Screenshot (185)
Screenshot (186)
Screenshot (187)
Screenshot (188)
Screenshot (189)
Screenshot (190)
Screenshot (191)
Screenshot (192)

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -1,10 +1,83 @@
...
\ No newline at end of file
+SELECT * FROM animals WHERE weight_kg<=17.3 AND weight_kg>=10.4;
+
+/* First Transation*/
+BEGIN WORK;
+ALTER TABLE animals 
+RENAME COLUMN species TO unspecified;
+SELECT * FROM animals;
+ROLLBACK;
+END;
```
Comment (V-Blaze) [inline queries.sql]: - You have done well so far :+1: :+1: But there are a few changes and requirements you need to meet before moving to the next milestone. :+1: :waffle: 

>Inside a transaction update the animals' table by setting the species column to unspecified. Verify that the change was made. Then roll back the change and verify that the species columns went back to the state before the transaction.

- [ ] It seems you didn't get the instructions clearly for this query. According to the project requirement, you are meant to set the value of the `species` column for all the animals to `unspecified`. You are not meant to rename the COLUMN but set the value of the data in the COLUMN. I suggest you update this query to include the two verifications. here's what it should look like. :+1: :+1: 

```sh
   BEGIN;
   UPDATE animals
   SET species = 'unspecified';
   SELECT species from animals; 
   ROLLBACK;
   SELECT species from animals;
```

**Review by V-Blaze — CHANGES_REQUESTED** [review]
## Hi @Graciano1997 

## Required changes 🖊️ 
# Almost there 🟡 🟦 🟨 

giphy


Great work so far, You have come a long way, keep moving forward👏 👍  
You've done well implementing the project requirements, but there are still some issues I think you need to improve on. 😄 👋 


 ## Highlights
- Correct Gitflow✔
- Most queries written correctly 👋 
-  Data inserted correctly into the database ✔

 ## Required Changes ♻️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: queries.sql (hunk 1 above)
Comment (Graciano1997) [reply queries.sql]: Upsss! thank  @V-Blaze

**Review by mikethreels — APPROVED** [review]
Hi Graciano, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- Good job following the instructions of the previous CR.
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**